common/EmpUtils.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
this util is super useful. There are a few places in the bots that could re-use this.